src/app/modules/shared_models/collectibles_entry.nim

```diff
@@ -320,6 +320,16 @@ QtObject:
     read = getTokenType
     notify = tokenTypeChanged
 
+  proc soulboundChanged*(self: CollectiblesEntry) {.signal.}
+  proc getSoulbound*(self: CollectiblesEntry): bool {.slot.} =
+    if not self.hasCollectibleData() or isNone(self.getCollectibleData().soulbound):
+      return true
```

Comment:
Do I read the code correctly that the collectible is `soulbound` by default?

Reply:
Indeed, in DB is `false`: https://github.com/status-im/status-go/blob/122490d8ee8c9646e260ac6c10956ffeaed585bb/walletdatabase/migrations/sql/1712567001_add_soulbound_collectible_cache.up.sql#L1